csfieldguide/chapters/content/en/computer-graphics/sections/graphics-transformations.md

```diff
@@ -313,7 +333,7 @@ This is called a *rotation*.
 In general, to rotate an image by a given angle you need to use the sine (abbreviated sin) and cosine (abbreviated cos) functions from trigonometry.
 You can use the interactive below to calculate values for the sin and cos functions.
 
-{interactive slug="trig-function-calculator" type="whole-page" text="Trig Function Calculator"}
+{interactive slug="trig-function-calculator" type="whole-page" alt="Trig Function Calculator"}
```

Comment:
I know you didn't create this interactive but it's part of the chapter and for some reason it shows up in my browser looking like this:
![image](https://user-images.githubusercontent.com/16066822/55691148-0d7f1980-59ef-11e9-94e4-0f3beb79adb3.png)

Reply:
That's as expected as far as I can see, what's wrong? I could make it in-page

Reply:
I fixed the layout